src/main/java/com/be3c/sysmetic/domain/strategy/controller/TraderStrategyController.java

```diff
@@ -37,7 +39,7 @@ public ResponseEntity<ApiResponse<Strategy>> insertStrategy(@Valid @RequestBody
                 .body(ApiResponse.success());
     }
 
-    // 전략명 중복
+    // 전략명 중복 확인
     @GetMapping("/strategy/duplication-name")
     public ResponseEntity<ApiResponse<Boolean>> checkDuplicationStrategyName(@RequestParam String name) {
```

Comment:
경로 매개변수를 쓰는 게 좋을 거 같습니다!
```suggestion
    @GetMapping("/strategy/duplication-name/{name}")
    public ResponseEntity<ApiResponse<Boolean>> checkDuplicationStrategyName(@PathVariable String name) {
```

Reply:
반영했습니다.